src/main/java/race/vehicle/Vehicles.java

```diff
@@ -0,0 +1,42 @@
+package race.vehicle;
+
+import race.Utils.RandomGenerator;
+import race.Utils.ResultUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Vehicles {
+    public static final String INITIAL_LOCATION = "-";
+    List<Car> list = new ArrayList<>();
```

Comment:
```suggestion
    private fianl List<Car> cars = new ArrayList<>();
```
1. 접근제어는 막아두는 것이 좋을 것 같아요 :)
2. 변수의 컬렉션을 네이밍할 때 변수명에 컬렉션의 타입을 명시하는 것은 지양하는 것이 좋아요.

[클린코드 2장. 의미 있는 이름 - 그릇된 정보를 피하라](https://github.com/Yooii-Studios/Clean-Code/blob/master/Chapter%2002%20-%20%EC%9D%98%EB%AF%B8%20%EC%9E%88%EB%8A%94%20%EC%9D%B4%EB%A6%84.md#%EA%B7%B8%EB%A6%87%EB%90%9C-%EC%A0%95%EB%B3%B4%EB%A5%BC-%ED%94%BC%ED%95%98%EB%9D%BC)

Reply:
접근제어자를 꼭 확인해야겠네요... 위에서 언급해주셨던 접근제어자 모두 수정하였습니다!